app/components/signature-pill.js

```diff
@@ -11,84 +11,48 @@ export default class SignaturePillComponent extends Component {
   @service intl;
   @service currentSession;
 
-  @tracked signingHubUrl;
-  @tracked isMarked = false;
-  @tracked isPrepared = false;
-  @tracked hasToBeApproved = false;
-  @tracked hasToBeSigned = false;
-  @tracked isSigned = false;
-  @tracked isRefused = false;
-  @tracked isCancelled = false;
-
-  constructor() {
-    super(...arguments);
-
-    this.loadData.perform();
-  }
+  status = trackedFunction(this, async () => {
+    const signMarkingActivity = await this.args.signMarkingActivity;
+    const signSubcase = await signMarkingActivity.signSubcase;
+    const signFlow = await signSubcase.signFlow;
```

Comment:
can `signSubcase` and `signFlow` be null? unsure.
A screenshot of error on ACC seems to have an occurence (could be old data)

Reply:
If there's a signMarkingActivity there should always be a `signSubcase` and `signFlow`. Anything else would idd be data inconsistency.

Reply:
> If there's a signMarkingActivity there should always be a `signSubcase` and `signFlow`. Anything else would idd be data inconsistency.

fine with me to keep this as-is